README.md

```diff
@@ -2,7 +2,7 @@
 
 Repository for ProjectE, a complete rewrite of EE2 (Equivalent Exchange 2) for modern Minecraft versions. Transmutation tables, collectors, condensers, flying rings, and all the other trinkets you love are here.
 
-Discover powerful alchemical tools, items, and devices. Break down unwanted items into EMC (Energy-Matter Covalence) and use that EMC to create new items.
+Discover powerful alchemical tools, items, and devices. Break down unwanted items into EMC (Energy-Matter Currency) and use that it to create new items.
```

Comment:
If changing "EMC" to "it" then you should remove the word "that". Given "use that it to create" does not make any sense.

Reply:
Oops, I have overlooked this